myFitnessTrainer/src/screens/ViewWorkoutPlanScreen.js

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import { StyleSheet, View, Text, ScrollView} from "react-native";
+import CreateNewPlanButton from "../components/CreateNewPlanButton";
+
+const ViewWorkoutPlanScreen = ({route}) => {
+    const { duration, fitnessGoal, fitnessLevel, startDate, daysCompleted, workoutsPerDay } = route.params;
+    const formattedStartDate = new Date(startDate).toDateString();
+    let formattedExercisesPerDay = []
+    for (const dayNum in workoutsPerDay){
+        formattedExercisesPerDay.push(<Text>Day {dayNum}{"\n"}</Text>);
+        for (const exerciseNum in workoutsPerDay[dayNum]){
+            formattedExercisesPerDay.push(<Text>{workoutsPerDay[dayNum][exerciseNum]}{"\n"}</Text>);
+        }
+        formattedExercisesPerDay.push(<Text>{"\n"}</Text>)
```

Comment:
```suggestion
    for (const dayNum in workoutsPerDay){
        formattedExercisesPerDay.push(<Text key={dayNum}>Day {dayNum}{"\n"}</Text>);
        for (const exerciseNum in workoutsPerDay[dayNum]){
            formattedExercisesPerDay.push(<Text key={dayNum.concat("-", exerciseNum)}>{workoutsPerDay[dayNum][exerciseNum]}{"\n"}</Text>);
        }
        formattedExercisesPerDay.push(<Text>{"\n"}</Text>)
```

Reply:
suggesting this update for a warning about items in a list not having unique keys.

Reply:
Thanks, good catch! Updated